backend-rust/src/graphql_api.rs

```diff
@@ -5105,24 +5152,33 @@ impl ContractInterrupted {
         .await?
         .ok_or(ApiError::NotFound)?;
 
-        let opt_event_schema = row
-            .display_schema
-            .as_ref()
-            .and_then(|schema| VersionedModuleSchema::new(schema, &None).ok())
-            .and_then(|versioned_schema| {
-                versioned_schema.get_event_schema(&row.contract_name).ok()
-            });
+        // Get the event schema if it exists.
+        let opt_event_schema = if let Some(event_schema) = row.display_schema.as_ref() {
+            let versioned_schema =
+                VersionedModuleSchema::new(event_schema, &None).map_err(|_| {
+                    ApiError::InternalError(
+                        "Database bytes should be a valid VersionedModuleSchema".to_string(),
```

Comment:
We should definitely also include this error to the logs.

Reply:
Yes, I agree. We have an issue for building a middleware to trace all `internalError` in our service (not implemented yet). When addressing the below issues we likely will decide to log/trace more information internally (e.g. exact database connection issues) while reporting only the parts of the internal errors to the user that don't leak any problematic information.
https://github.com/Concordium/concordium-scan/issues/270